### ISSUE TITLE
NVDA concatenates text and does not read alerts in Microsoft Edge's "About" page

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#19140]
### Brief summary

When we open the Microsoft Edge's **About** page, the browser sends an alert that it is checking for updates and if there are such or not. Also, when we read that **About** page in browse mode, each line that is visually on a new line is treated by NVDA as a new line.

Not anymore though. Since a few versions of Microsoft Edge (when the **About** page was changed a bit, both the alerts and the splitting of the new lines got broken.

Now NVDA does not read any alerts on that page and the new lines in each section of the page are concatenated in one single line when reading in browse mode.

I haven't checked about the alerts, but the reading of the separate lines works OK with Narrator. That made me think why is so. And then I figured it out - Narrator uses UIA to access Microsoft Edge, while NVDA primarily uses Iaccessible2 (if I'm not mistaking). When I configured NVDA to also use only UIA for Edge, NVDA was also reading the new lines separately as it should.

Actually, I just checked - both Narrator and NVDA do not read the alerts even with UIA, so they are probably broken for that page by Edge itself.

I don't know about the split lines though - if the fault is in NVDA or Edge.

### Steps to reproduce

1. Launch NVDA.
2. Open Microsoft Edge.
3. From Microsoft Edge's menu, go to **Help and Feedback** -> **About Microsoft Edge**.
4. When the **About** page loads, observe if NVDA will report that Edge is checking for updates.
5. Switch NVDA to browse mode and then read the text in the **About** page.
6. Observe if NVDA reads as separate lines or joined lines the text that is visually on separate lines (e.g. the labels for options and their descriptions - they visually are on separate lines).
7. From NVDA's advanced settings, set NVDA to use only UIA for accessing Chromium-based browsers and apply the changes.
8. Return to Microsoft Edge's **About** page and repeat steps 5 and 6.

### Actual behavior

- NVDA does not read any alerts in the **About** page.
- While reading, NVDA joins / concatenates lines that are visually on separate lines.

### Expected behavior

- NVDA reads any alerts in the **About** page.
- While reading, NVDA splits to separate lines lines that are visually on separate lines.

### NVDA type

installed copy

### NVDA version

alpha-53202,9e37e579 (2026.1.0.53202)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Yes - 2025.3 and the result is the same.

### Windows version

Windows 11 Pro version 25H2 (build 26200.6899), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".

### Name and version of other software in use when reproducing the issue

Microsoft Edge version 141.0.3537.99 (Official Build) (64-bit)

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have not run the COM Registration Fixing Tool

**Comment by SaschaCowley** [comment]
Hi @k-kolev1985, Please report this issue to Microsoft.

**Comment by k-kolev1985** [comment]
Hi @SaschaCowley 

Can you recommend an appropriate channel to do so? Preferably somewhere that I can track the reported issue?

**Comment by k-kolev1985** [comment]
@msftedad 
Can you recommend a channel where I can report this to the Edge dev team? Thanks. 😊🙏

**Comment by seanbudd** [comment]
Sorry for the late reply, `alt+shift+l` from edge should open the feedback dialog. There's also the [accessibility desk](https://www.microsoft.com/en-au/accessibility/disability-answer-desk)

**Comment by k-kolev1985** [comment]
@seanbudd 
Thanks. I filed a report via the Edge's feedback mechanism (the "Send Feedback" button from its toolbar).  I hope the issue will get addressed soon.
